constitution.md

```diff
@@ -0,0 +1,232 @@
+# Chisel Working Group (CWG) Provisional Constitution
+
+### Table of Contents
+* [Relationship to CHIPS Alliance](#relationship-to-chips-alliance)
+* [Mission Statement](#mission-statement)
+* [Members](#members)
+  * [Technical Advisory Committee](#technical-advisory-committee)
+  * [Administrator](#administrator)
+  * [Reviewers](#reviewers)
+  * [Developers](#developers)
+  * [Volunteers](#volunteers)
+* [Collaboration Policy](#collaboration-policy)
+  * [Communication Channels](#communication-channels)
+  * [Development Policy](#development-policy)
+* [Amending the Constitution](#amending-the-constitution)
+* [TAC Voting](#tac-voting)
+* [Code of Conduct (Contributors)](#code-of-conduct-contributors)
+  * [Our Pledge](#our-pledge)
+  * [Our Standards](#our-standards)
+  * [Our Responsibilities](#our-responsibilities)
+  * [Scope](#scope)
+  * [Enforcement](#enforcement)
+  
+## Relationship to CHIPS Alliance
+
+The Chisel Working Group is a member of CHIPS Alliance and subject to the bylaws established by
+the CHIPS Alliance Board.
+
+## Mission Statement
+The Chisel ecosystem is a stable, user-friendly platform for production-quality RTL generator development, 
+designed to support the needs of a variety of RTL development communities, including industry, academia, and individuals.
+
+It includes support for the “Chisel stack”, including but not limited to FIRRTL, Testers, Chisel Utilities, documentation, 
+and administrative work.
+
+## Members
+The Chisel self-governance consists of five types of members:
+**Technical Advisory Committee members**, **administrators**, **reviewers**, **developers**,
+and **volunteers**.
+
+### Technical Advisory Committee
+The role of Technical Advisory Committee (TAC) is to represent the interests of the
+projects and members of the Chisel Working Group. This includes, but is not limited
+to, facilitating communication and collaboration between members, as well as interfacing
+with CHIPS Alliance on behalf of the Chisel community. Additionally, the TAC is
+responsible for upholding the policies outlined in this document.
+
+Other responsibilities include: admitting (and if necessary expelling) projects,
+organizing community meetings/conferences, resolving disputes between projects,
+allocating funds, clarifying procedures for each membership type, and publicizing
+the Chisel community.
```

Comment:
Should delegation be explicitly called out as a responsibility? For example, it could be the TAC's role to delegate some of it's responsibilities to a sub-committee they select. I suppose delegation is more of an ability than it is a responsibility, but it seems like something that would be good to mention somewhere.

Reply:
I think delegation is implicit in any leadership position.